### ISSUE TITLE
Define what other content will be included in "Latest Updates"

### ISSUE THREAD

**Issue by noahmanger** [18F/fec-cms#524]
The goal here is to further flesh out the concept for the Latest Updates page by identifying all the different types of content that should be included. 

The FEC currently has a "What's new" page that serves as a feed of a bunch of different types of updates. They're interested in replicating that functionality here, but we should work through:
- A) Does that make sense? Is that something we should do?
- B) What else might be useful to include?

This will likely take some back and forth with the partners, but the task here is just to agree on a list. We can tackle design refinements in subsequent issues.

**Completion criteria:**
- [ ] A list of agreed upon items that will ultimately populate this page

**Comment by nickykrause** [comment]
Current "What's New" page, for reference: http://fec.gov/general/whatsnew.shtml

**Comment by nickykrause** [comment]
Things that are currently on the What's New page:
- Press Releases (already in our latest updates feed)
- Weekly Digests (ditto)
- The Record monthly releases (ditto)
- Audit Reports (link to [pages like this one](http://fec.gov/audits/2012/AuditReport_2014_BrendaLawrenceForCongress.shtml))

- AO Documents
    - Draft AO announcements ([PDFs](http://saos.fec.gov/aodocs/201610.pdf))
    - AO Requests ([PDFs](http://saos.fec.gov/aodocs/201615R_1.pdf))
    - Withdrawals of AO requests ([PDFs](http://saos.fec.gov/aodocs/201605W_1.pdf))
    - Closeout letters for AO requests ([PDFs](http://saos.fec.gov/aodocs/2016-04.pdf))

- OIG documents
    - IG Statements ([PDF](http://fec.gov/fecig/documents/ManagementandPerformanceChallenges-2015-FinalReport.pdf))
    - OIG semi-annual reports to congress ([PDF](http://fec.gov/fecig/documents/OIGSemiannualReporttoCongress-May2016-FinalPublicDistribution.pdf))
    - OIG studies ([PDF](http://fec.gov/fecig/documents/RootCausesofLowEmployeeMoraleStudy-FinalReport-OIG-15-06.pdf))
    - OIG annual work plans ([PDF](http://fec.gov/fecig/documents/OIGFY2016WorkPlan-FINAL-10-01-15_002.pdf))
- Comments received on proposals under consideration ([links to pages like this] (http://fec.gov/law/policy.shtml#formsrevisions))

- The yearly edition of the FEC's regulations. ([links to pages like this](http://fec.gov/law/cfr/cfr.shtml))

**Comment by emileighoutlaw** [comment]
This is awesome, Nicky! Adding to your list, here are the things (in addition to what's currently in `What's New`) that we'd also like to consider for `Latest updates`

- Tips for Treasurers
- Meeting agendas
- Meeting notices
- Sunshine notices

**Comment by nickykrause** [comment]
I'm looking at this question of what else should be included in the latest updates section. So far, I've made a list of what I found to be the latest updates over the past year or so on the existing What's New page. (See above for that list and a link to What's New).

There are some things I don't understand about this task, so here are a few questions:
- @noahmanger (cc: others who have thoughts): How urgent is this? Are we just trying to get existing content moved over, since this section is going live soon, or are we taking a bigger look at the purpose of Latest Updates and being reflective about what could/should be included? 
    - I ask because, at first glance, I don't have specific concerns with replicating what they have already included, but that may be because I am lacking insight into what users would expect from this section. We could start by just moving over what they have, at the least, and then thinking through whether or not additional items should be added (but I am not sure if we have any info that would suggest what else we should add--i.e., what kinds of "updates" are users seeking? What do they expect to see here?)

- @emileighoutlaw: If we add more types of content, am I correct in assuming that the changes will affect this text at the top of the page?: 
<img width="797" alt="screen shot 2016-09-29 at 2 17 12 pm" src="https://cloud.githubusercontent.com/assets/1450173/18968601/48af8b14-864f-11e6-9be4-24224977e607.png">

- @jenniferthibault: There are some existing content items that, if we do include them in Latest Updates, appear to link to pages for which we may or may not have visual design solutions. Specifically:
    - the "Audit Reports" in the "What's New" feed link to [pages like this](http://fec.gov/audits/2012/AuditReport_2014_BrendaLawrenceForCongress.shtml). 
    - the comments received on proposals under consideration links to [pages like this](http://fec.gov/law/policy.shtml#formsrevisions)
    - the yearly edition of the FEC's regulations links to [a page like this](http://fec.gov/law/cfr/cfr.shtml). 

Perhaps some of these pages are familiar to you? I am not sure what kinds of pages are designed and what aren't. 

Thanks for helping me fill in the gaps. We can briefly sync up if that makes sense. Just trying to gather all the info at the moment.

**Comment by noahmanger** [comment]
Good question! And apologies, this totally slipped through my "post-meeting-issue-flesh-out" work. The goal here is, as you put it, to take a bigger look and be reflective about what all should be here. We'll do the first release with just press releases and weekly digests, but the FEC is curious about what all we envision belonging here.

**Comment by noahmanger** [comment]
And I will now update the issue description to that effect :)

**Comment by emileighoutlaw** [comment]
@nickykrause — You're totally correct that it will impact the content on the page, and those changes will depend on what we choose to add.

**Comment by nickykrause** [comment]
@emileighoutlaw: Thanks for the list of additional items! I just saw that after I posted my comment. Where did that list come from? Just wondering if those items were already discussed at length or something. I see no problem with including them, but, then again, I don't really know much about what some of them are (i.e., tips for treasurers). 😄 

@noahmanger Thanks!

To all: As for taking a bigger look at what it might make sense to put in this section, I am trying to figure out what the criteria for inclusion should be. 

All I have come up with so far is "frequency" and "timeliness" -- meaning, content that is released on some periodic basis, and which may be immediately relevant to people (such that they would want to be notified of that content's "release"). 

Other than that, I am not immediately aware of what commonalities might exist across the content that _currently_ makes its way into "What's New." The things in there now seem to be varied, but certainly all "periodic" or "timely."

If our criteria for inclusion are as simple as that, then it seems like a pretty straightforward exercise to think through what else might be periodic or timely.

**Comment by emileighoutlaw** [comment]
Hey hey hey! Tips for treasurers are a public-facing publication, so we identified it in our initial designs of this section as something to include. 

The other meetings things are things Amy K. has talked about bringing over. 


My thinking about this starts to get muddy when I try to imagine including AO documents and OIG things to the section. I'm having trouble putting into words why those things feel different to me than everything else in your list, but they do.

**Comment by noahmanger** [comment]
I think I share @emileighoutlaw 's feelings about including both types of content in the same feed. Essentially, we're talking about two different categories:
- **Communications published by the agency:** These are the "missives" (as originally conceived by @onezerojeremy ), which essentially are the agency's way of "talking to" the world. Pres
- **Notifications about agency activity:** These are the notifications about AOs, OIG reports, and other agency activity. 

And some things seem to fall somewhat in between, like Sunshine Notices or meeting agendas or notices. 

Both categories share in common that they're a way to get a timely update on agency activity, but they're different in their substance. 

So I think that part of this task is to actually refine what criteria we're using to determine what gets included. Should this page simply house all of the agency's broadcast communication out to the world, or should it also include notifications? If it includes notifications, should it include other notifications, like filing deadlines or when new data has been loaded (hypothetical question)? If it doesn't, where do those live, if anywhere?

---

As a side note on the topic,  I found this in the memo about agency outreach: 
> Include MURs, ADRS and AFs disclosed in the Enforcement Query System under the "What's New" section of the website. (If people don't know how to search for new cases, they will find out about newly resolved matters only on Fridays, with the publication of the Weekly Digest.)

**Comment by noahmanger** [comment]
To give this some data, I'm looking at the existing What's New page in Analytics:
![image](https://cloud.githubusercontent.com/assets/1696495/18976468/db9690e6-8666-11e6-9cd8-d662b6c0a13a.png)
![image](https://cloud.githubusercontent.com/assets/1696495/18976479/e6c5cd2e-8666-11e6-8d26-abed1ba4cc46.png)

This is just since July 1, but I'm seeing a few things:
- Not many people seem to use it. Out of 933 unique visitors since 7/1, the most heavily clicked links are some press releases that drew 13 clicks
- Almost all of the clicks are to press releases and digests, and a few to audit reports. The AOs seem virtually untouched
- There's _very_ few clicks earlier than 7/1, suggesting that people do really use this for finding recent stuff

Not entirely sure what all to do with that info, but there ya go.

**Comment by nickykrause** [comment]
@noahmanger:

Totally agree with this statement:

> So I think that part of this task is to actually refine what criteria we're using to determine what gets included.

...and many of the other things you covered in your posts. 

---

So, I'm going to assemble what we have so far regarding **possible inclusion criteria**, between our two posts so far:

- Timely / periodic content
- Communications published by the agency
- Communication about agency activity 

---

and here is the **current list of possible items for inclusion**, all in one place:

- Press Releases (already in our latest updates feed)
- Weekly Digests (ditto)
- The Record monthly releases (ditto)
- Audit Reports (link to [pages like this one](http://fec.gov/audits/2012/AuditReport_2014_BrendaLawrenceForCongress.shtml))
- AO Documents
    - Draft AO announcements ([PDFs](http://saos.fec.gov/aodocs/201610.pdf))
    - AO Requests ([PDFs](http://saos.fec.gov/aodocs/201615R_1.pdf))
    - Withdrawals of AO requests ([PDFs](http://saos.fec.gov/aodocs/201605W_1.pdf))
    - Closeout letters for AO requests ([PDFs](http://saos.fec.gov/aodocs/2016-04.pdf))
- OIG documents
    - IG Statements ([PDF](http://fec.gov/fecig/documents/ManagementandPerformanceChallenges-2015-FinalReport.pdf))
    - OIG semi-annual reports to congress ([PDF](http://fec.gov/fecig/documents/OIGSemiannualReporttoCongress-May2016-FinalPublicDistribution.pdf))
    - OIG studies ([PDF](http://fec.gov/fecig/documents/RootCausesofLowEmployeeMoraleStudy-FinalReport-OIG-15-06.pdf))
    - OIG annual work plans ([PDF](http://fec.gov/fecig/documents/OIGFY2016WorkPlan-FINAL-10-01-15_002.pdf))
- Comments received on proposals under consideration ([links to pages like this] (http://fec.gov/law/policy.shtml#formsrevisions))
- The yearly edition of the FEC's regulations. ([links to pages like this](http://fec.gov/law/cfr/cfr.shtml))
- MURs, ADRS and AFs disclosed in the Enforcement Query system
- Tips for Treasurers
- Meeting agendas
- Meeting notices
- Sunshine notices

---

Some thoughts/questions:

I agree that it feels a bit weird to include both "communications by the agency" and "communications about agency activity" in the same list, which leads me to a few initial thoughts:
- **First**: We could include only the things that are communications by the agency, since we have already started that precedent with this section (press releases, weekly digests, etc.). That idea begs the question of what to do with the other "updates." (Do we create a separate "feed" of some kind, for those kinds of updates? If so: insert all the subsequent questions here about where it would live, what it contains, etc.)
- **Second**: Does it make sense to keep them in the same feed, even though they are different "types" of things, given that we do have filtering mechanisms in place on the Latest Updates page?:

<img width="541" alt="screen shot 2016-09-30 at 9 22 01 am" src="https://cloud.githubusercontent.com/assets/1450173/18994933/a30e77b4-86ef-11e6-99ba-a2ebeb9dcb74.png">

@emileighoutlaw I am curious what you think!

**Comment by jenniferthibault** [comment]
@nickykrause sorry for delay, but yes, many of these new items are new patterns (or new combinations of existing patterns, but I haven't started assessing)

**Comment by emileighoutlaw** [comment]
This is nuance, but by definition _updates_ are for “bringing (a person, organization, etc.) up to date on a particular subject.” 

To me, that indicates that the information in an “updates” section isn’t intended to be the whole hog. As a user, I should be able to find out a little bit about everything that’s going on at the agency, but not all of everything going on at the agency. And I should have clear pathways to learn more about anything that strikes my fancy in that section.

Keeping that in mind, Latest updates feels like the place to find:
- Weekly Digests
- Press releases
- The Record
- Tips for treasurers (time-pegged, not news-pegged, but that still makes sense here)

Audits and OIG reports are covered in Weekly Digests. They will also live as full-text documents in “About the FEC.” AOs, MURs, ADRs, and AFs are also covered in Weekly Digests. They also will live as full-text documents in “Legal resources.” I can see a strong case for not including any of these things in the “Latest updates” section.

The only thing that leaves, that I don’t see a clear home for are meeting agendas, meeting notices, and Sunshine notices. Weekly digests do include blurbs about upcoming meetings. But there isn’t a clear place in our IA for these documents to live, if not in “Latest updates.”

**tldr:** That's a longwinded way of saying I think we should include only the things that are communications by the agency, and I don't think that means we need to create another feed. But we would need to find a space for meetings notices and agendas.

**Comment by emileighoutlaw** [comment]
Here's another article that I think could inform our work on this: http://gadgetopia.com/post/7622

My takeaways:

Static content (the tree) is for Known Unknowns: 
> _This is information that you know exists, but that you do not know the value of. I know that there is an abbreviation for the element Boron. This information exists somewhere, but I do not know the value of it. I would have to go looking for this, but the important fact is that I know it’s out there somewhere._

Latest updates is for Unknown Unknowns: 
>_This is information that you don’t know exists, and (obviously) don’t know the value of. My eight-year-old daughter doesn’t know the abbreviation for Boron either. The difference between her and I is that she doesn’t even know that this piece of information exists.  It’s a crucial difference._

>..._How could I go searching for any of this information? I can’t, because I don’t know the information exists. In every case, I would eventually find out, but that would be sub-optimal. I would eventually find out about the bus bearing down on me at 50 m.p.h. too, but the goal would to find out early enough to step out of the way._

> ... While [people] will gladly go climb a tree to find something they know is there, they need to be dropped into a river of news in order to find out the things they don’t know.

**Comment by jenniferthibault** [comment]
You nailed a thing I wanted to think though @emileighoutlaw:

Latest + Updates. Not "all + documents"

Thoughts on some of the unknowns:

- To me, meeting agendas live as links from their calendar events. I don't think they need to be collected anywhere else, as long as you can sort the calendar down to "open meetings" and click in to get to the agenda. If there is a way to see all of them, it feels like a separate index that you get to _from another agenda_ only.

  
<img width="950" alt="screen shot 2016-10-05 at 2 24 30 pm" src="https://cloud.githubusercontent.com/assets/11636908/19126053/5ec10240-8b07-11e6-9b1a-22d7c76f4b84.png">


- Meeting notices....I'm not sure. If an event exists in the calendar, does it really need to be announced separately? And if it does, could it be announced _in_ the weekly digests?

- Sunshine notices.... I guess I could see these going in the latest updates section. I don't really understand them though—are sunshine notices essentially the same thing as meeting notices?

**Comment by nickykrause** [comment]
@emileighoutlaw: 👍 to everything you outlined
@jenniferthibault: I agree on both counts re: the meeting agendas and meeting notices. I also second your points about not really knowing what sunshine notices are. 

@emileighoutlaw: Do you have any clarification about the sunshine notices?

**Comment by emileighoutlaw** [comment]
Yes! When we talk about the Sunshine Act, we're talking about The Government in the Sunshine Act, which is pro government transparency. 

The Act requires that all federal government meetings be open to the public (with only a few exceptions).

Sunshine Act notices are announcements that _before a meeting_ that says a meeting is scheduled and generally what's on the docket for discussion. 

So every time the Commission meets, FEC.gov publishes:
- A [Sunshine Act notice](http://www.fec.gov/sunshine/2016/open/notice20160714.pdf)
- [Draft meeting minutes](http://www.fec.gov/agenda/2016/documents/mtgdoc_16-36-a.pdf)
- [Approved meeting minutes](http://www.fec.gov/agenda/2016/documents/approved_16-36-a.pdf)
- A [unique HTML page](http://www.fec.gov/agenda/2016/agenda20160616.shtml) with audio files, captions, and supporting documents (like draft advisory opinions, rulemaking documents, Commissioner statements, etc.)

**Comment by noahmanger** [comment]
So much good thinking here. 

I think, given the purpose of the Sunshine Act Notices, that they might make sense in the Latest Updates feed, as they're a communication published by the agency about activity at the agency. 

All in all, I feel good about a concept for this page that focuses exclusively on it as a feed of communications published by the agency and think we have a good rationale for it.

**Comment by nickykrause** [comment]
Thanks, @emileighoutlaw! And, @noahmanger, I agree. 

So, here is a summary of all the things and their respective fates:

**Latest updates:**
- Press Releases (already there)
- Weekly Digests (already there)
- The Record monthly releases (already there)
- Tips for Treasurers (future)
- Sunshine notices (future)


**Already in Weekly Digests:**
- Audit Reports
- AO Documents
- OIG documents
- MURs, ADRS and AFs disclosed in the Enforcement Query system


**Calendar:**
- Meeting agendas


**Unclear?** :thinking: 
- Meeting notices
   - Possibly discontinuing these? Is it necessary to output a "meeting notice," in addition to a Sunshine Notice, and a meeting agenda?
- Comments received on proposals under consideration ([links to pages like this] (http://fec.gov/law/policy.shtml#formsrevisions))
    - Anyone know if this kind of thing already appears in the Weekly Digests? (Even if it doesn't at this point, does it seem related enough to the legal docs that it could appear there?)
    - One thing we do know: This wouldn't go in `Latest Updates`, since it doesn't fit the framework of timely communications published by the agency.
- The yearly edition of the FEC's regulations. ([links to pages like this](http://fec.gov/law/cfr/cfr.shtml))
    - Again, would this appear in the digests anyway? This does seem like a communication by the agency, though...except that it doesn't happen frequently, and may therefore not be "timely" enough?

**Comment by noahmanger** [comment]
Re: the Unclear items:
- Does anyone have an example of a meeting notice? I can't find what these even are.
- Agree that comments received wouldn't fall here
- The yearly regulations also feel like they don't fall into this category, and are more of a "new document" than a timely communication

I think we have a good plan here, but we'll need to discuss it with the FEC. This feels like a task to pick up with the new content manager team, so I propose we set this aside for the time being and discuss it with them next week.

**Comment by emileighoutlaw** [comment]
👍 to talking about this with content managers. 

To wrap up a couple outsanding questions: 
- I do believe the Weekly Digest covers yearly releases of the FEC's regulations 
- Meeting notices and Sunshine notices are the same thing

**Comment by noahmanger** [comment]
Next task:
- [ ] @emileighoutlaw discuss this with content manager group as part of a broader meeting

**Comment by emileighoutlaw** [comment]
Update: Content managers can't get together to discuss this until after the election on Tuesday.

**Comment by emileighoutlaw** [comment]
We'll be discussing this with content managers in person next Thursday

**Comment by emileighoutlaw** [comment]
Jen and I just paired with Amy on this. Our thoughts:

The intention of `Latest updates` is to publish a feed of content that was created specifically  _for users_. Therefore the feed should include:
- Press releases
- Weekly Digests
- FEC Record articles
- Tips for Treasurers
- Sunshine notices (notices that a meeting is going to happen and generally what will be discussed at that meeting)
- Meeting agendas (published after Sunshine notices and including more detail)


Everything else we were considering including in the feed (new AOs, IG reports, etc.) is either _already covered_ by one of the above (Weekly Digests, for example) or can be surfaced in the "miscellaneous items" section on the homepage, if there's a pressing need.

Thoughts anyone? 💟

**Comment by noahmanger** [comment]
Sounds like a great plan.

**Comment by emileighoutlaw** [comment]
Does this sound good to you, too @nickykrause ? If so, I'll close and open a new issue for design and implementation.

**Comment by nickykrause** [comment]
Yes!

**Comment by emileighoutlaw** [comment]
🎉
